src/components/pages/AdminClearingHouseSD/AdminClearingHouseSDElements.tsx

```diff
@@ -0,0 +1,329 @@
+/********************************************************************************
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+import { useEffect, useState } from 'react'
+import './AdminClearingHouseSD.scss'
+import { Trans } from 'react-i18next'
+import { t } from 'i18next'
+import {
+  Button,
+  CircleProgress,
+  ToggleSwitch,
+  Typography,
+} from '@catena-x/portal-shared-components'
+import {
+  type ComapnyDataType,
+  useFetchCompanyDataQuery,
+  useFetchConnectorsQuery,
+  useTriggerCompanyDataMutation,
+  useTriggerConnectorsMutation,
+  PAGE_SIZE,
+} from 'features/adminClearingHouseSD/adminClearingHouseSDApiSlice'
+import { error } from 'services/NotifyService'
+import { getRequireHttpsUrlPattern } from 'services/EnvironmentService'
```

Comment:
as discussed, you still need to change to the proper value before merging

Reply:
Changes done

Reply:
It's not fully enabled with your changes, @oyo wanted to look into it